### ISSUE TITLE
IdeaVim:  eb doesn't select the current word

### ISSUE THREAD

**Issue by sdoerner** [chtenb/helix.vim#3]
In Helix, you can use `eb` to select the current word. This doesn't seem to work in `helix.idea.vim` (didn't test other editor configs). Instead, it selects the first character of the word, and the character before that.

**Comment by chtenb** [comment]
Yes, see my comment in the Idea PR: https://github.com/chtenb/helix.vim/pull/2#discussion_r1787227861
The vanilla motions I wrote in helix.vim don't appear to work in IdeaVim.
I cannot test this because I do not use IdeaVim IDEs, and the author of the PR did not figure out how to implement the word motions.
If you want to take a stab at properly implementing these motions for IdeaVim, that would be great.

**Comment by sdoerner** [comment]
It seems like ]` (jump to next lowercase mark) does not work in IdeaVim, same with "previous lowercase mark".

I tried just replacing it with `a but that doesn't work. The feature seems to be needed to handle some edge cases wrt. whitespace.

For now, I filed a bug with IdeaVim to add support for this motion: https://youtrack.jetbrains.com/issue/VIM-3731/Add-support-for-jump-to-previous-next-lowercase-mark

**Comment by ristomatti** [comment]
Great to see other IDEA users here as well! I quite literally took a jump in the deep end with this since I've never been an advanced Vim user (even though I used it ~10 years as the default shell editor).

**Comment by ristomatti** [comment]
@sdoerner Perhaps update the issue title to e.g "IdeaVim: eb doesn't select the current word" as IdeaVim support was contributed by me and does not match 1:1 to the great work done by @chtenb for Vim itself.

**Comment by sdoerner** [comment]
I fixed the upstream bug and sent them a [PR](https://github.com/JetBrains/ideavim/pull/1068). After that, we need to remove some visual mode overrides (#4).

PR #4 is in draft mode so far, as the upstream PR isn't merged or released yet.

If anyone is up for building IdeaVim from source, please let me know how this works for you.

**Comment by ristomatti** [comment]
> If anyone is up for building IdeaVim from source, please let me know how this works for you.

I'd love to but sadly I'm too busy with other stuff for the coming months.

**Comment by mbouron** [comment]
I've built IdeaVim (latest master as of `1105f61988f4e5d63e370b791e200e3058f8d282` + your fix applied on top) + https://github.com/chtenb/helix.vim/pull/4 and now e/b/w motion work as expected. Thanks a lot !
